uobtheatre/bookings/schema.py

```diff
@@ -22,6 +22,11 @@ class Meta:
 
 
 class TicketNode(DjangoObjectType):
```

Comment:
I think we need to add so auth to the checked_in_by field. I cant remember if there is a nice way of doing that but might just be easiest to do something like
```python
def resolve_checked_in_by(self):
     if () # remind me to try and write this tomorrow 😂
```